Products/CMFPlone/resources/browser/styles.py

```diff
@@ -143,6 +152,18 @@ def styles(self):
                     'bundle': 'diazo'}
 
             result.append(data)
+
+        # custom.css
+        custom_css = {
+            'rel': 'stylesheet',
+            'conditionalcomment': '',
+            'src': "{0}/custom.css?timestamp={1}".format(
+                self.site_url,
+                self.custom_css_timestamp,
+            ),
+            'bundle': 'custom-css'
+        }
+        result.append(custom_css)
```

Comment:
I still think that this should be added only if not empty (see https://github.com/plone/Products.CMFPlone/pull/3089#discussion_r423717288) but we can do this refinement later.

Reply:
I agree.